src/shapes/sprite.js

```diff
@@ -14,16 +14,18 @@ function SpriteShape(params) {
     dx: params.x,
     dy: params.y,
     speed: properties.speed,
-    animation: null
+    update: null,
+    animation: properties.animation === true ? true : false,
```

Comment:
In cases like this, doesn't matter if is a **real** boolean, if use set/or/create animation params will be true